tests/backbones/test_backbones.py

```diff
@@ -0,0 +1,27 @@
+from mantisshrimp.backbones import *
+import pytest
+import torch
+
+
+def test_torchvision_backbones():
+    supported_backbones = [
```

Comment:
If we define the supported backbones as a class method, we don't have to redefine them here again 😃

Reply:
Defined as a class method with self inside mantis_faster_rcnn